pkg/services/mysql/dbms_types.go

```diff
@@ -13,17 +18,16 @@ type firewallRule struct {
 	EndIP   string `json:"endIPAddress"`
 }
 
-func (
-	d *dbmsManager,
-) getProvisionParametersSchema() map[string]service.ParameterSchema {
-	return getDBMSCommonProvisionParamSchema()
-}
-
 type dbmsInstanceDetails struct {
 	ARMDeploymentName        string `json:"armDeployment"`
 	ServerName               string `json:"server"`
 	FullyQualifiedDomainName string `json:"fullyQualifiedDomainName"`
 	EnforceSSL               bool   `json:"enforceSSL"`
+	HardwareFamily           string `json:"hardwareFamily"`
+	Cores                    int    `json:"cores"`
+	Storage                  int    `json:"storage"`
+	BackupRetention          int    `json:"backupRetention"`
+	BackupRedundancy         string `json:"backupRedundancy"`
```

Comment:
> Added them for later upgrade use so we can do validation of the update request vs what was there before

We can validate the update request against the original provisioning parameters if we need to since we hang on to them. I guess really, I should reframe my comment-- we don't need two copies of this info.

Reply:
Yeah good point. And we can compare to previous update parameters for any subsequent update operations.

Reply:
> And we can compare to previous update parameters for any subsequent update operations.

When an update is complete, imo, the updating parameters should completely replace the original provisioning parameters so that the instance then appears as it if was originally provisioned in its current state. Or at least this has been my plan.